src/main/java/nl/novi/Les11SpringBootTechItEasyApplication/controllers/TelevisionsController.java

```diff
@@ -60,18 +58,16 @@ public ResponseEntity<Object> updateTelevision(@PathVariable Long id, @RequestBo
         updateTelevision.setOriginalStock(television.getOriginalStock());
         updateTelevision.setSold(television.getSold());
 
-        televisionRepository.save(updateTelevision);
+        televisionService.saveTelevision(updateTelevision);
```

Comment:
Ik zie dat je heel veel logica in de controller aan het toepassen bent, maar dit hoort in de service. De controller moet je dom houden.

Reply:
Vandaag was heel veel bezig geweest en alles reeds naar de Service verplaatst ;)